kaal-domain/src/main/kotlin/cz/eman/kaal/domain/result/ApiErrorResult.kt

```diff
@@ -0,0 +1,11 @@
+package cz.eman.kaal.domain.result
+
+/**
+ *
+ * @author Roman Holomek <roman.holomek@eman.cz>
+ */
+data class ApiErrorResult(
+    val errorCode: ErrorCode,
+    val errorMessage: String? = null,
+    val apiThrowable: Throwable? = null
+) : ErrorResult(errorCode, errorMessage, apiThrowable)
```

Comment:
Better solution with elimination duplicate properties
```
data class ApiErrorResult(
    override val code: ErrorCode,
    override val message: String? = null,
    override val throwable: Throwable? = null
) : ErrorResult(code, message, throwable)
```

Reply:
Thx for hint,,sorry copy & paste from a real project